Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityToolkit.Services.InputSimulation.asmdef

```diff
@@ -5,7 +5,8 @@
     ],
     "optionalUnityReferences": [],
     "includePlatforms": [
-        "Editor"
+        "Editor",
+        "WSA"
```

Comment:
This should ease #3971 on WSA, but we'll still need a proper fix for all platforms.